packages/ag-charts-website/src/pages/sitemap.astro

```diff
@@ -17,7 +17,7 @@ const parsedSitemap = parseSitemap(xmlSitemap);
     <div class:list={['contentViewport layout-grid']}>
         <div className={'layout-max-width-small'}>
             <h1>AG Charts Sitemap</h1>
-            <Sitemap client:only="react" sitemap={parsedSitemap} />
+            <Sitemap client:only="react" sitemap={parsedSitemap} site={'Charts'} />
```

Comment:
What does `site={'Charts'}` do? It doesn't seem like the `Sitemap` component supports it? Unless it's supposed to be implemented here?

Reply:
It's implemented in ag-website-shared on the grid side (pending merge), just needs syncing over: https://github.com/ag-grid/ag-grid/pull/8532/commits/4863cba705a8b6bfaaf68040b101c6e51aeac4b1

Reply:
Ah cool, maybe to improve SEO, this could be updated to use `client:load` too, like grid: https://github.com/ag-grid/ag-grid/blob/latest/documentation/ag-grid-docs/src/pages/sitemap.astro#L18

Reply:
Good shout, pushed that change